src/transformers/models/mask2former/modeling_mask2former.py

```diff
@@ -2416,7 +2416,7 @@ def forward(
         >>> masks_queries_logits = outputs.masks_queries_logits
 
         >>> # Perform post-processing to get instance segmentation map
-        >>> pred_instance_map = image_processor.post_process_semantic_segmentation(
+        >>> pred_instance_map = image_processor.post_process_instance_segmentation(
         ...     outputs, target_sizes=[image.size[::-1]]
```

Comment:
It would be also nice to include!
```suggestion
        ...     outputs, target_sizes=[(image.height, image.width)]
```

Reply:
Should we change this line in the examples for all three segmentation tasks?